src/gromo/containers/growing_graph_network.py

```diff
@@ -763,16 +720,16 @@ def choose_growth_best_action(
 
         # Reconstruct graph
         best_option = options[best_ind]
-        del options
-
-        self.dag = copy.copy(best_option["dag"])
-        self.growth_history = best_option["growth_history"]
-        self.growth_loss_train = best_option["loss_train"]
-        self.growth_loss_dev = best_option["loss_dev"]
-        self.growth_loss_val = best_option["loss_val"]
-        self.growth_acc_train = best_option["acc_train"]
-        self.growth_acc_dev = best_option["acc_dev"]
-        self.growth_acc_val = best_option["acc_val"]
+        del options  # TODO: memory freed?
```

Comment:
What is the interest of this del? Are options stored represent a large amount of memory?

Reply:
The options also store different versions of the GrowingDAG so a deletion is probably needed